src/app/components/Account/AccountLink.tsx

```diff
@@ -26,15 +24,9 @@ const WithTypographyAndLink: FC<{
   const to = RouteUtils.getAccountRoute(scope, address)
   return (
     <WithHoverHighlighting address={address}>
-      <Typography variant="mono" component="span" sx={{ display: 'inline-flex' }}>
-        {labelOnly ? (
-          children
-        ) : (
-          <Link component={RouterLink} to={to} sx={{ display: 'inline-flex' }}>
-            {children}
-          </Link>
-        )}
-      </Typography>
+      <span className="inline-flex font-medium text-primary">
+        {labelOnly ? children : <RouterLink to={to}>{children}</RouterLink>}
```

Comment:
This is different than prev `{labelOnly ?...` condition. Is this expected or we should keep prev logic ? In other places you keep diff color for labelOnly